VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Main_Module Torque_Vector Driveline Precharge Dashboard bootloader TESTER Precharge BMS1 TEST_NODE TEST_NODE_2 DAQ


BO_ 2214592577 driver_request: 4 Main_Module
 SG_ regen : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ accel : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214598913 main_status: 3 Main_Module
 SG_ precharge_state : 16|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ apps_state : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ car_state : 0|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214592578 torque_request: 6 Torque_Vector
 SG_ rear_right : 36|12@1+ (1,0) [0|0] "" Vector__XXX
 SG_ rear_left : 24|12@1+ (1,0) [0|0] "" Vector__XXX
 SG_ front_right : 12|12@1+ (1,0) [0|0] "" Vector__XXX
 SG_ front_left : 0|12@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2147490050 bitstream_flash_status: 1 Torque_Vector
 SG_ flash_timeout_rx : 1|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ flash_success : 0|1@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214592515 front_wheel_data: 8 Driveline
 SG_ right_normal : 48|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left_normal : 32|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ right_speed : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left_speed : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214592579 rear_wheel_data: 8 Driveline
 SG_ right_normal : 48|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left_normal : 32|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ right_speed : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left_speed : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2348810883 front_motor_currents: 4 Driveline
 SG_ right : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2348810947 rear_motor_currents: 4 Driveline
 SG_ right : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ left : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2281734148 test_precharge_msg: 1 Precharge
 SG_ test_precharge_sig : 0|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028613 raw_throttle_brake: 8 Dashboard
 SG_ brake1 : 48|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ brake0 : 32|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ throttle1 : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ throttle0 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214592517 start_button: 1 Dashboard
 SG_ start : 0|1@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2415925566 bitstream_data: 8 TESTER
 SG_ d7 : 56|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d6 : 48|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d5 : 40|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d4 : 32|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d3 : 24|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d2 : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d1 : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ d0 : 0|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2415925630 bitstream_request: 5 TESTER
 SG_ download_size : 1|32@1+ (1,0) [0|0] "" Vector__XXX
 SG_ download_request : 0|1@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2148123710 mainmodule_bl_cmd: 2 TESTER
 SG_ data : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ cmd : 0|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2148123774 dashboard_bl_cmd: 2 TESTER
 SG_ data : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ cmd : 0|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214624554 balance_request: 2 Precharge
 SG_ volts : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2214624555 soc1: 2 BMS1
 SG_ soc : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028095 test_msg: 2 TEST_NODE
 SG_ test_sig : 0|16@1- (1,0) [0|0] "" Vector__XXX

BO_ 2483028159 test_msg2: 2 TEST_NODE
 SG_ test_sig2 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028223 test_msg3: 2 TEST_NODE
 SG_ test_sig3 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028287 test_msg4: 2 TEST_NODE
 SG_ test_sig4 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028351 test_msg5: 2 TEST_NODE
 SG_ test_sig5 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2348810751 wheel_speeds: 8 TEST_NODE
 SG_ right_speed : 32|32@1- (1,0) [0|0] "kph" Vector__XXX
 SG_ left_speed : 0|32@1- (1,0) [0|0] "kph" Vector__XXX

BO_ 2348826559 adc_values: 5 TEST_NODE
 SG_ pot3 : 24|12@1+ (1,0) [0|0] "" Vector__XXX
 SG_ pot2 : 12|12@1+ (1,0) [0|0] "" Vector__XXX
 SG_ pot1 : 0|12@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2550136831 daq_response_TEST_NODE: 8 TEST_NODE
 SG_ daq_response : 0|64@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028093 test_msg_2: 2 TEST_NODE_2
 SG_ test_sig : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028157 test_msg2_2: 2 TEST_NODE_2
 SG_ test_sig2 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028221 test_msg3_2: 2 TEST_NODE_2
 SG_ test_sig3 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028285 test_msg4_2: 2 TEST_NODE_2
 SG_ test_sig4 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483028349 test_msg5_2: 8 TEST_NODE_2
 SG_ test_sig5_3 : 32|32@1- (1,0) [0|0] "" Vector__XXX
 SG_ test_sig5_2 : 16|16@1- (1,0) [0|0] "" Vector__XXX
 SG_ test_sig5 : 0|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2550136829 daq_response_TEST_NODE_2: 8 TEST_NODE_2
 SG_ daq_response : 0|64@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483032050 daq_command_TEST_NODE: 8 DAQ
 SG_ daq_command : 0|64@1+ (1,0) [0|0] "" Vector__XXX

BO_ 2483031922 daq_command_TEST_NODE_2: 8 DAQ
 SG_ daq_command : 0|64@1+ (1,0) [0|0] "" Vector__XXX




CM_ BU_ Main_Module "";
CM_ BU_ Torque_Vector "";
CM_ BU_ Driveline "";
CM_ BU_ Precharge "";
CM_ BU_ Dashboard "";
CM_ BU_ bootloader "";
CM_ BU_ TESTER "";
CM_ BU_ Precharge "";
CM_ BU_ BMS1 "";
CM_ BU_ TEST_NODE "";
CM_ BU_ TEST_NODE_2 "";
CM_ BU_ DAQ "";
CM_ BO_ 2214592577 "Driver throttle/regeneration request";
CM_ SG_ 2214592577 regen "regen request. 0-4096";
CM_ SG_ 2214592577 accel "accel request. 0-4096";
CM_ BO_ 2214598913 "Status message for main module";
CM_ SG_ 2214598913 precharge_state "Precharge complete state";
CM_ SG_ 2214598913 apps_state "Accelerator plausibility state";
CM_ SG_ 2214598913 car_state "Main car state";
CM_ BO_ 2214592578 "Torque request for all motors.";
CM_ SG_ 2214592578 rear_right "torque request. 0-4096";
CM_ SG_ 2214592578 rear_left "torque request. 0-4096";
CM_ SG_ 2214592578 front_right "torque request. 0-4096";
CM_ SG_ 2214592578 front_left "torque request. 0-4096";
CM_ BO_ 2147490050 "Status message for bitstream flashing";
CM_ SG_ 2147490050 flash_timeout_rx "Timeout from CAN rx";
CM_ SG_ 2147490050 flash_success "Done flashing bitstream";
CM_ BO_ 2214592515 "Wheel data for the rear tires";
CM_ SG_ 2214592515 right_normal "";
CM_ SG_ 2214592515 left_normal "";
CM_ SG_ 2214592515 right_speed "";
CM_ SG_ 2214592515 left_speed "";
CM_ BO_ 2214592579 "Wheel data for the rear tires";
CM_ SG_ 2214592579 right_normal "";
CM_ SG_ 2214592579 left_normal "";
CM_ SG_ 2214592579 right_speed "";
CM_ SG_ 2214592579 left_speed "";
CM_ BO_ 2348810883 "DC Current of the motor controllers";
CM_ SG_ 2348810883 right "";
CM_ SG_ 2348810883 left "";
CM_ BO_ 2348810947 "DC Current of the motor controllers";
CM_ SG_ 2348810947 right "";
CM_ SG_ 2348810947 left "";
CM_ BO_ 2281734148 "test precharge desc";
CM_ SG_ 2281734148 test_precharge_sig "";
CM_ BO_ 2483028613 "throttle_brake desc";
CM_ SG_ 2483028613 brake1 "";
CM_ SG_ 2483028613 brake0 "";
CM_ SG_ 2483028613 throttle1 "";
CM_ SG_ 2483028613 throttle0 "";
CM_ BO_ 2214592517 "Car start button pressed";
CM_ SG_ 2214592517 start "";
CM_ BO_ 2415925566 "Bitstream download command";
CM_ SG_ 2415925566 d7 "Bitstream data word 7";
CM_ SG_ 2415925566 d6 "Bitstream data word 6";
CM_ SG_ 2415925566 d5 "Bitstream data word 5";
CM_ SG_ 2415925566 d4 "Bitstream data word 4";
CM_ SG_ 2415925566 d3 "Bitstream data word 3";
CM_ SG_ 2415925566 d2 "Bitstream data word 2";
CM_ SG_ 2415925566 d1 "Bitstream data word 1";
CM_ SG_ 2415925566 d0 "Bitstream data word 0";
CM_ BO_ 2415925630 "Bitstream state command";
CM_ SG_ 2415925630 download_size "size for expected bitstream download (bytes)";
CM_ SG_ 2415925630 download_request "Request to download new bitstream";
CM_ BO_ 2148123710 "Bootloader command for main module";
CM_ SG_ 2148123710 data "Bootloader data";
CM_ SG_ 2148123710 cmd "Bootloader Command type";
CM_ BO_ 2148123774 "Bootloader command for main module";
CM_ SG_ 2148123774 data "Bootloader data";
CM_ SG_ 2148123774 cmd "Bootloader Command type";
CM_ BO_ 2214624554 "Request BMSs to discharge their cells to the desired voltage";
CM_ SG_ 2214624554 volts "";
CM_ BO_ 2214624555 "Estimated soc";
CM_ SG_ 2214624555 soc "";
CM_ BO_ 2483028095 "test_msg desc";
CM_ SG_ 2483028095 test_sig "";
CM_ BO_ 2483028159 "test_msg2 desc";
CM_ SG_ 2483028159 test_sig2 "";
CM_ BO_ 2483028223 "test_msg2 desc";
CM_ SG_ 2483028223 test_sig3 "";
CM_ BO_ 2483028287 "test_msg4 desc";
CM_ SG_ 2483028287 test_sig4 "";
CM_ BO_ 2483028351 "test_msg5 desc";
CM_ SG_ 2483028351 test_sig5 "";
CM_ BO_ 2348810751 "wheel speeds";
CM_ SG_ 2348810751 right_speed "";
CM_ SG_ 2348810751 left_speed "";
CM_ BO_ 2348826559 "adc potentiometer readings";
CM_ SG_ 2348826559 pot3 "";
CM_ SG_ 2348826559 pot2 "";
CM_ SG_ 2348826559 pot1 "";
CM_ BO_ 2550136831 "daq response from node TEST_NODE";
CM_ SG_ 2550136831 daq_response "";
CM_ BO_ 2483028093 "test_msg desc";
CM_ SG_ 2483028093 test_sig "";
CM_ BO_ 2483028157 "test_msg2 desc";
CM_ SG_ 2483028157 test_sig2 "";
CM_ BO_ 2483028221 "test_msg2 desc";
CM_ SG_ 2483028221 test_sig3 "";
CM_ BO_ 2483028285 "test_msg4 desc";
CM_ SG_ 2483028285 test_sig4 "";
CM_ BO_ 2483028349 "test_msg5 desc";
CM_ SG_ 2483028349 test_sig5_3 "";
CM_ SG_ 2483028349 test_sig5_2 "";
CM_ SG_ 2483028349 test_sig5 "";
CM_ BO_ 2550136829 "daq response from node TEST_NODE_2";
CM_ SG_ 2550136829 daq_response "";
CM_ BO_ 2483032050 "daq command for node TEST_NODE";
CM_ SG_ 2483032050 daq_command "";
CM_ BO_ 2483031922 "daq command for node TEST_NODE_2";
CM_ SG_ 2483031922 daq_command "";




SIG_VALTYPE_ 2348810751 left_speed : 1;
SIG_VALTYPE_ 2348810751 right_speed : 1;
SIG_VALTYPE_ 2483028349 test_sig5_3 : 1;


